tests/pom.xml

```diff
@@ -415,9 +415,14 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
       </activation>
       <properties>
         <hono.mongodb.disabled>true</hono.mongodb.disabled>
-        <hono.deviceregistry.image>hono-service-device-registry-file</hono.deviceregistry.image>
-        <hono.deviceregistry.resources.folder>deviceregistry</hono.deviceregistry.resources.folder>
+        <hono.postgres.disabled>true</hono.postgres.disabled>
+        <hono.deviceregistry.image>hono-service-device-registry-jdbc</hono.deviceregistry.image>
```

Comment:
What about renaming the profile "device-registry-file" to something like "device-registry-embedded" or "device-registry-embedded-db"? The old name still suggests the file-based implementation is used. As far as I can tell, the profile is not referenced anywhere in the project, so the impact should be minimal anyway.

Reply:
The profile name should correspond to the property that activates it. Can we change the value of "hono.deviceregistry.type" to "embedded"?

Reply:
I deliberately kept the original value in order to not require any configuration changes in other places like the CI build definition. The data is still being persisted to the the local file system so I thought it to (still) be an appropriate name. The name of the profile is also only relevant within the Hono build process so I do not think that it has much of an overall impact anyway.